conductor/server/workflows/decommission_device.json

```diff
@@ -0,0 +1,54 @@
+{
+    "name": "decommission_device",
+    "description": "Removes device info from all services.",
+    "version": 1,
+    "tasks": [
```

Comment:
I think it should be possible to refactor these to FORK/JOIN with requests to respective service running in parallel.